### PR TITLE
Format SMS posession proving form

### PR DESCRIPTION
**Pull request by relaxnow**
https://www.pivotaltracker.com/n/projects/1163646/stories/100299518 punt 4

screen shot 2016-06-13 at 16 21 57

### CODE REVIEW

**Comment by rjkip** [comment]
Welk punt gaat dit om?

**Comment by relaxnow** [comment]
@rjkip punt 4

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/provePossession.html.twig [hunk 1]
```diff
@@ -20,6 +20,26 @@
 
     <hr>
 
-    {{ form(form) }}
+    {{ form_start(form)|replace({'form-horizontal': ''})|raw }}
```
Comment (rjkip) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/provePossession.html.twig]: Does `form_start(form, {'horizontal': false})` work?

Comment (relaxnow) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/Registration/Sms/provePossession.html.twig]: Ah I copy & pasted that from another template but don't need it. In fact removing it fixes the vertical align bug on the label. 🎉

**Comment by relaxnow** [comment]
screen shot 2016-06-13 at 17 16 44

**Comment by rjkip** [comment]
:ok_hand:

**Pull request merged**